src/core/component/directives/attrs/test/b-component-emitter-dummy/b-component-emitter-dummy.styl

```
@@ -0,0 +1,15 @@
+/*!
+ * V4Fire Client Core
+ * https://github.com/V4Fire/Client
+ *
+ * Released under the MIT license
+ * https://github.com/V4Fire/Client/blob/master/LICENSE
+ */
+
+@import "components/super/i-block/i-block.styl"
+
+$p = {
+
+}
+
+b-component-emitter-dummy extends i-block
```

В префикс дамми компонента стараемся включать путь до модуля, чтобы было понятно к какому модулю это относится, в данном случае надо сделать префикс `b-component-directives-*`

```suggestion
b-component-directives-emitter-dummy extends i-block
```